ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml

```diff
@@ -26,82 +26,57 @@ tests:
   steps:
     cluster_profile: gcp-qe
     env:
-      WORKER_REPLICA_COUNT: "24"
-    post:
-    - chain: ipi-gcp-post
-    pre:
-    - chain: ipi-gcp-pre
-    - chain: create-infra-move-ingress-monitoring-registry
-    - chain: workers-scale
-    workflow: openshift-qe-control-plane
+      COMPUTE_NODE_REPLICAS: "24"
```

Comment:
I'd be interested to see how consistently the installer can install 24 nodes, might consider moving to using `workers-scale` if failures start occuring

Reply:
Ack, triggered a job 🤞

Reply:
The failure was related to cluster-install, and not to `COMPUTE_NODE_REPLICAS`.
Triggered a new job

Reply:
Hi @paigerube14 👋 
The latest control-plane tests have passed, can you ptal?